test/unit/ui/selenium/LoginTests.js

```diff
@@ -0,0 +1,265 @@
+const { Assertion } = require('chai');
+const { assert } = require('chai');
+const {Builder, By} = require('selenium-webdriver');
+const HomePage = require('../../../../src/POM/HomePage');
+const LoginPage = require('../../../../src/POM/LoginPage');
+const readline = require('readline');
+const email = process.env.LOGIN_EMAIL;
+const password = process.env.LOGIN_PASSWORD;
+const adminEmail = process.env.LOGIN_EMAIL_ADMIN;
+const adminPassword = process.env.LOGIN_PASSWORD_ADMIN;
+require('dotenv').config();
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+
```

Comment:
This file should be in test/ui/selenium/LoginTest

Reply:
done